pkg/wolfssl/Makefile.dep

```diff
@@ -82,6 +82,10 @@ ifneq (,$(filter wolfcrypt_random,$(USEMODULE)))
   USEMODULE += random
 endif
 
+ifneq (,$(filter wolfcrypt_cryptocell,$(USEMODULE)))
+  SRC +=  tfm.c
+endif
+
```

Comment:
Shouldn't this go in `pkg/wolfssl/Makefile.wolfcrypt` file?

Reply:
Not really sure, it could be.

Reply:
Done.